examples/python/sourcepkg/build.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+pip3 install -r ${SRC_PKG}/requirements.txt -t ${SRC_PKG} && cp -r ${SRC_PKG} ${DEPLOY_PKG}
```

Comment:
It's a *great* idea to let users provide their own build scripts! 

However, we should provide good defaults too. 

In other words, the python builder should contain a default build.sh that's invoked when user does not specify it. That default build script should basically be this file -- pip install + copy to destination.

Reply:
A default build command will be packed into python env. 
https://github.com/life1347/fission/blob/cf30d9514ce850ee1b54fdaa6eb4bd029655e2a3/environments/python3/builder/Dockerfile#L8
The default build command will be specified when a package is created without specified a build command.
https://github.com/life1347/fission/blob/cf30d9514ce850ee1b54fdaa6eb4bd029655e2a3/fission/function.go#L149